tools/csv-merger/csv-merger.go

```diff
@@ -749,6 +750,12 @@ func injectCSVRelatedImages(r *unstructured.Unstructured) error {
 			"image": *noobaaDBContainerImage,
 		})
 	}
+	if *ocsMustGatherImage != "" {
+		relatedImages = append(relatedImages, map[string]interface{}{
+			"name":  "ocs-must-gather-image",
```

Comment:
This is a bit inconsistent with the others images. When following the pattern, we would do:
```suggestion
			"name":  "ocs-must-gather",
```